### ISSUE TITLE
The storageFormat property of Security_Log_Profile does not support spaces in string

### ISSUE THREAD

**Issue by rnotley** [F5Networks/f5-appsvcs-extension#747]
### Environment
 * Application Services Version: 3.43.0
 * BIG-IP Version: 15.1.8.1

### Summary
When sending an AS3 declaration for a network Security_Log_Profile, if the storageFormat property is a user-defined string that contains one or more spaces, AS3 returns an error.

### Steps To Reproduce
1. Submit the following declaration:
```json
{
  "schemaVersion": "3.43.0",
  "class": "ADC",
  "myTenant": {
    "class": "Tenant",
    "myApplication": {
      "class": "Application",
      "network_log_profile": {
        "class": "Security_Log_Profile",
        "network": {
          "logRuleMatchAccepts": true,
          "storageFormat": "${date_time} ${bigip_hostname} ${acl_policy_name},${acl_policy_type},${acl_rule_name},${action},${context_name},${context_type},${dest_ip},${dest_port},${drop_reason},${protocol},${sa_translation_pool},${sa_translation_type},${src_ip},${src_port},${translated_dest_ip},${translated_dest_port},${translated_ip_protocol},${translated_src_ip},${translated_src_port},${translated_vlan},${vlan}"
        }
      }
    }
  }
}
```

2. Observe the following error response:
```json
{
  "code": 422,
  "host": "localhost",
  "message": "declaration failed",
  "response": "\\\"${bigip_hostname}\\\" unknown property",
  "runTime": 1906,
  "tenant": "myTenant"
}
```

### Expected Behavior
AS3 should be able to handle spaces in the storageFormat string.  When editing the string in the Web UI, spaces are accepted and the log format includes spaces as well.

### Actual Behavior
AS3 declaration fails

### Work-around
None

**Comment by sunitharonan** [comment]
Thank you.  In order to prioritize and understand the issue better, please reach out to us at automation_toolchain_pm@f5.com

**Comment by sunitharonan** [comment]
Closing as this has been released in AS3 3.48.0
